.github/workflows/release.yml

```diff
@@ -274,8 +295,6 @@ jobs:
         with:
           name: contrast-releases.json
           path: ./packages/contrast-releases.json
-      - name: Clean up git tree
-        run: git clean -fx :/
```

Comment:
Can we instead modify this statement to only reset checked-in files and keep the build results?